src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -98,6 +98,19 @@ public function setStyle($name, OutputFormatterStyleInterface $style)
     }
 
     /**
+     * Adds a new style.
+     *
+     * @param string      $name       The style name
+     * @param string|null $foreground The style foreground color name
+     * @param string|null $background The style background color name
+     * @param array       $options    The style options
+     */
+    public function addStyle($name, $foreground = null, $background = null, array $options = array())
```

Comment:
Looks confusing with the `setStyle()` above. Both methods does the same thing, but with different arguments. I would prefer to keep `setStyle()` for both methods instead of adding a new one.